.lift.toml

```diff
@@ -0,0 +1 @@
+disableTools=["refactor-first"]
```

Comment:
To prove this works, you should revert this change (merge the other change). And create a new PR with this in, to show that the warning isn't made.

Or would that not work, do you have to merge it first? Would be good to verify it does what we want in some way.

Reply:
Yeah unfortunately I have to merge it first as I the config file needs to be on dev branch to take effect. I'll open a test PR after merging this one to confirm comment isn't posted, but based on the resolved sonatype-lift comment above, it looks like it will work. The link in PR desc. is very appropriate for our exact situation too, so I'm fairly confident